cf_units/etc/site.cfg.template

```diff
@@ -1,2 +0,0 @@
-[System]
```

Comment:
Technically, this is still valid (XML file, not ``.so``).

Reply:
Oops, yep! (I guess I need to fix our conda recipe too.)